ingq/book/infra/pagination/cursor.py

```
@@ -0,0 +1,114 @@
+            if "updated_at" not in cursor_dict or "book_id" not in cursor_dict:
+                raise OrderStrategyAndCursorParameterMismatchException(
+                    "UpdatedAt을 정렬 전략으로 선택하셨습니다. 커서 값으로 updated_at과 book_id가 있는지 확인해주세요."
+                )
+            try:
+                updated_at = datetime.fromisoformat(cursor_dict["updated_at"])
+                book_id = int(cursor_dict["book_id"])
+                return UpdatedAtCursor(updated_at=updated_at, book_id=book_id)
+            except (ValueError, TypeError):
+                raise OrderStrategyAndCursorParameterMismatchException(
+                    "ISO 형식의 시간이 맞는지, book_id가 int인지 확인해주세요."
+                )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)